integration/src/test/resources/collector_config/stdout/do-not-track.hocon

```diff
@@ -0,0 +1,88 @@
+collector {
+  enableStartupChecks = false
+
+  interface = 0.0.0.0
+  port = 12345
+
+  paths { }
+
+  p3p {
+    policyRef = "/w3c/p3p.xml"
+    CP = "NOI DSP COR NID PSA OUR IND COM NAV STA"
+  }
+
+  crossDomain {
+    enabled = false
+    domains = [ "*" ]
+    secure = true
+  }
+
+  cookie {
+    enabled = true
+    expiration = "365 days"
+    name = "sp"
+    secure = false
+    httpOnly = false
+  }
+
+  doNotTrackCookie {
+    enabled = true
+    name = "dnt"
+    value = "dnt"
+  }
+
+  cookieBounce {
+    enabled = false
+    name = "n3pc"
+    fallbackNetworkUserId = "00000000-0000-4000-A000-000000000000"
```

Comment:
Do we really need to specify all the config parameters of the collector for each test ? We have defaults for almost all of them, can't we just specify the ones that are interesting for us for each test ?